scripts/checkpoint_converters/convert_llama_nemo_to_hf.py

```diff
@@ -138,7 +142,8 @@ def convert(input_nemo_file, output_hf_file, precision=None, cpu_only=False) ->
     ffn_hidden_size = model.cfg.ffn_hidden_size
     num_query_groups = model.cfg.get("num_query_groups", head_num)  # different num_query_groups for 70B
 
-    head_size = hidden_size // head_num
+    # Use kv_channels if available. Otherwise, use hidden_size // head_num as head_size
+    head_size = model.cfg.get("kv_channels", hidden_size // head_num)
```

Comment:
I think in nemo config `kv_channels` can be set to `null` explictly, which would cause issue here. I would suggest changing to something like:
```
if model.cfg.get("kv_channels", None) is None:
    head_size = hidden_size // head_num
else:
    head_size = model.cfg.kv_channels
```

Reply:
@suiyoubi  Thank you for the suggestion! Revised accordingly.